Feature/reporting blogpost (#152)
morloy/ledgy.com: src/markdown/updates/kpis-and-reports.mdx

---
title: "KPIs & Reports"

> description: "Keep track of your KPIs and integrate your recurring reports on Ledgy"

Reviewer: ```suggestion
description: "Keep track of your KPIs and share your recurring reports through Ledgy"
```

images:
  - KPI-values.png
  - KPI-chart.png
  - report.png
---

As a **startup founder**, or as its **investor**, you have most likely experienced the pain of consistently staying up to date with the company progress and its activities.

As a founder, you know how tedious it is to write recurrent reports, to keep up with KPIs, and to have to constantly inform all your stakeholders about it.

As an investor, you want to know how the company is doing at all times, and appreciate proper and transparent communication.

> ###### Suffer no more, Ledgy brings to you KPI tracking and recurring reports

Reviewer: ```suggestion
###### Now on Ledgy: KPI tracking and recurring reports
```

<Img src="KPI-values.png" align="left" size="300"/>

<br />

Integrate your key performance indicator values on Ledgy in a matter of minutes, copying and pasting existing values if needed.

Ledgy will display beautiful graphs that you will be able to share with your investors and integrate into your reports.

<Img src="KPI-chart.png"/>

<br />

You can track as many activities as you want, whether it’s revenue, planted trees, or rockets successfully landed on Mars.

**Once your KPIs are up and running, it will be time to implement your recurring reports** 🚀

<Img src="report.png" />

> The awesomeness doesn’t stop here. With Ledgy Reports, you can also

Reviewer: ```suggestion
With Ledgy Reports, you can also
```

Author: I saw this one coming 😜

* Create **templates** and use the same structure for every recurring report, which will save you time and keep your documents consistent
* **Share** the report with your selected stakeholders by clicking on *Publish*

###### Ready to try it out?

[Click here](https://app.ledgy.com) to go straight to the Ledgy app

> At Ledgy, we are always striving to thrive and bring the best solutions for your company and your investors. If you have any suggestions, don’t hesitate to ​[contact us​](mailto:contact@ledgy.com).

Reviewer: ```suggestion
At Ledgy, we are always striving to help founders with all aspects around equity management and investor relations. If you have any suggestions, don’t hesitate to ​[contact us​](mailto:contact@ledgy.com).
```
